packages/vscode-extension/src/utilities/subprocess.ts

```diff
@@ -84,7 +84,10 @@ export function execSync(name: string, args?: string[], options?: execa.SyncOpti
   return result;
 }
 
-export function command(commandWithArgs: string, options?: execa.Options) {
+export function command(
+  commandWithArgs: string,
+  options?: execa.Options & { silentErrorsOnExit?: boolean }
```

Comment:
This option name was confusing – "silent" is used as a verb and we don't ignore exit messages, we ignore all messages. Maybe typical `quiet` option would be more fitting?